airflow/www/views.py

```diff
@@ -4877,22 +4877,14 @@ class CustomUserDBModelView(MultiResourceUserMixin, UserDBModelView):
 class CustomUserLDAPModelView(MultiResourceUserMixin, UserLDAPModelView):
     """Customize permission names for FAB's builtin UserLDAPModelView."""
 
-    pass
-
```

Comment:
This would be resulted in EOF while parsing
isn't it?

Reply:
Yep :)